src/extension.ts

```diff
@@ -334,6 +316,18 @@ export function activate(context: vscode.ExtensionContext): void {
     });
 
     registerEvent('appService.fileEditor.onDidSaveTextDocument', vscode.workspace.onDidSaveTextDocument, async function (this: IActionContext, doc: vscode.TextDocument): Promise<void> { await fileEditor.onDidSaveTextDocument(this, context.globalState, doc); });
+    registerCommand('appService.EnableFileLogging', async (node?: IAzureNode<SiteTreeItem> | IAzureNode<FolderTreeItem> | IAzureParentNode<IAzureTreeItem>) => {
```

Comment:
Do you need `| IAzureParentNode<IAzureTreeItem>`? Feels like the other two describe all entry points

Reply:
Without `IAzureParentNode<IAzureTreeItem>`, I was getting type conflictions because of line 326.  Specifically, it wouldn't let me cast node.parent to IAzureNode<SiteTreeItem>.